Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -107,8 +109,7 @@ public async Task<bool> FindCloudlet()
                 throw new FindCloudletException("Last RegisterClient was unsuccessful. Call RegisterClient again before FindCloudlet");
             }
 
-            location = GetLocationFromDevice();
-            UpdateCarrierName();
+            await UpdateLocationAndCarrierInfo();
```

Comment:
I am a big fan of consolidation, However a small reminder the [next feature we ll have](https://mobiledgex.atlassian.net/browse/EDGECLOUD-3157) exposing Location Parameter in FindCloudlet.
UpdateLocationAndCarrierInfo(latitude, longtiude) VS UpdateLocation(latitude, longtiude) + UpdateCarrierName()

Reply:
That feature shouldn't affect the consolidation. I think that feature will just override the UpdateLocation call if the developer provides a long/lat